src/Google.Api.Gax/project.json

```diff
@@ -7,6 +7,9 @@
     "projectUrl": "htttps://github.com/googleapis/gax-dotnet",
     "licenseUrl": "https://developers.google.com/open-source/licenses/bsd",
     "tags": [ "Google" ],
+    "compilationOptions": {
+        "keyFile": "../../Gax.snk"

```

Comment:
Unless I'm missing something, this ../../Gax.snk file doesn't seem to exist in this repo?